src/mlpack/tests/io_test.cpp

```diff
@@ -785,11 +803,12 @@ BOOST_AUTO_TEST_CASE(DoubleParamTest)
 
   ParseCommandLine(argc, const_cast<char**>(argv));
 
-  BOOST_REQUIRE(IO::HasParam("double"));
-  BOOST_REQUIRE_CLOSE(IO::GetParam<double>("double"), 3.12, 1e-10);
+  REQUIRE(IO::HasParam("double"));
+  REQUIRE(IO::GetParam<double>("double") == Approx(3.12).epsilon(1e-12));
 }
 
-BOOST_AUTO_TEST_CASE(RequiredOptionTest)
+TEST_CASE_METHOD(IOTestDestroyer, "RequiredOptionTest",
+                "[IOTest]")
```

Comment:
```suggestion
TEST_CASE_METHOD(IOTestDestroyer, "RequiredOptionTest", "[IOTest]")
```

Looks like this would fit into the same line.